packages/flutter/test/material/tab_bar_theme_test.dart

```diff
@@ -20,7 +20,7 @@ const List<Tab> _tabs = <Tab>[
   Tab(text: _tab3Text, icon: Icon(Icons.looks_3)),
 ];
 
-final TabController _tabController = TabController(length: 3, vsync: const TestVSync());
+TabController get _tabController => TabController(length: 3, vsync: const TestVSync());
```

Comment:
This is still a little odd. Better to put `TabController(length: _tabs.length, vsync: const TestVSync())` in _buildTabBar() and the one test that uses this getter, put _buildTabBar() in _withTheme()

In other words, get rid of get _tabController and _buildTabBar().

Reply:
Ah, I see now, done!